projects/pluto/src/main/java/nl/fairspace/pluto/config/dto/PlutoConfig.java

```diff
@@ -44,11 +46,25 @@ public static class Storage {
         private String rootDirectoryIri;
     }
 
+    @Data
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public static class Service {
```

Comment:
I would name it more specifically, something like ExternalService, otherwise it is a bit general

Reply:
It is not only about external services, but I can try different name... This is how we call this option in the configuration, just `services`, and I just moved it from Saturn to Pluto.